source/GameAction.cpp

```diff
@@ -392,9 +410,21 @@ GameAction GameAction::Instantiate(map<string, string> &subs, int jumps, int pay
 
 	for(auto &&it : giftShips)
 		result.giftShips.emplace_back(it.first, !it.second.empty() ? it.second : GameData::Phrases().Get("civilian")->Get());
-	result.giftOutfits = giftOutfits;
 
-	result.payment = payment + (jumps + 1) * payload * paymentMultiplier;
+	// Add any additional required outfits
+	map<const Outfit *, int> mergedGiftedOutfits(giftOutfits);
+	int outfitObjectiveCost = 0;
+	for(auto &it : outfitObjective)
+	{
+		outfitObjectiveCost += it.first->Cost() * it.second * OutfitBulkBonus(it.second);
+
+		if(mergedGiftedOutfits.count(it.first) == 0)
+			mergedGiftedOutfits[it.first] = 0;
+		mergedGiftedOutfits[it.first] += -1 * it.second;
```

Comment:
```suggestion
		mergedGiftedOutfits[it.first] -= it.second;
```

Reply:
If its not negative you will end up gaining that many outfits.

Reply:
What I modified is the exact same thing as what you did, just written differently

Reply:
Oh didn’t see the - just the =